native/python/zds_py.hpp

```diff
@@ -0,0 +1,23 @@
+#ifndef ZDS_PY_HPP
+#define ZDS_PY_HPP
+
+#include <string>
+#include <vector>
+#include "../c/zdstype.h"
+#include "../c/zds.hpp"
+
+void create_dataset(std::string dsn, DS_ATTRIBUTES attributes);
+
+std::vector<ZDSEntry> list_datasets(std::string dsn);
+
+std::string read_dataset(std::string dsn, std::string codepage = "");
+
+std::string write_dataset(std::string dsn, std::string data, std::string codepage = "", std::string etag = "");
+
+void delete_dataset(std::string dsn);
+
+void create_member(std::string dsn);
+
+std::vector<ZDSMem> list_members(std::string dsn);
```

Comment:
(question) If these functions are transformed by SWIG, is it possible to add doc for them here so that it gets captured in the generated Python code? Or does it need documented separately for consumers of this Python SDK?